i18n/locales/nl-NL/common.json

```diff
@@ -0,0 +1,565 @@
+{
+  " - Confirm transaction on your {{deviceModel}}": " - Bevestig de transaction op je {{deviceModel}}",
+  " LSK": " LSK",
+  " Make sure that you are using the latest version of Lisk.": " Controleer of je de laatste versie van Lisk gebruikt.",
+  " is out. ": " is uit. ",
+  "% of successfully forged blocks in relation to total blocks that were available for this particular delegate to forge": "% van succesvol gesmede blokken in relatie tot het totaal aantal blokken die beschikbaar waren om te smeden voor de afgevaardigde",
```

Comment:
`gesmede` sounds a bit much like `smithing` 🗡 
Maybe better to use here the original `forged` as it's like a term associated with Lisk.